lineage/states/stateCommon.py

```diff
@@ -103,3 +103,52 @@ def basic_censor(cell):
                 cell.get_sister().left.observed = False
                 cell.get_sister().right.observed = False
 
+def negative_LL_atonce(x, uncens_obs, uncens_gammas, cens_obs, cens_gammas):
+    """ uses the negative_LL_atonce and passes the vector of scales and the shared shape parameter. """
+    return negative_LL_sep_atonce(x[1:5], x[0], uncens_obs, uncens_gammas, cens_obs, cens_gammas)
+
+def negative_LL_sep_atonce(scales, a, uncens_obs, uncens_gammas, cens_obs, cens_gammas):
```

Comment:
Can this not call `negative_LL_sep` to reuse code better?

Reply:
done.

Reply:
I had to return the code back because of a dimension mismatch that I had to fix in the `negative_LL_sep_atonce()`.

Reply:
Ok, we can always simplify the code later.